compiler/cpp/src/parser/ast/BooleanExprAST.h

```diff
@@ -0,0 +1,18 @@
+//
+// Created by Marc on 20.05.2021.
+//
+
+#ifndef COMPILER_BOOLEANEXPRAST_H
```

Comment:
This is an include guard, right? You could use `#pragma once` instead, couldn't you?

Reply:
Yes. Didn't know about `#pragma once`. Thanks for the tip